app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -1224,6 +1218,17 @@ class HomeFragment : Fragment() {
         }
     }
 
+    private fun displayWallpaper() {
```

Comment:
nit: maybe name this `displayWallpaperIfEnabled`, or move the check to above before calling this function.